providers/implementations/ciphers/cipher_aes_ccm.c

```diff
@@ -24,8 +29,7 @@ static void *aes_ccm_newctx(void *provctx, size_t keybits)
 {
     PROV_AES_CCM_CTX *ctx;
 
-    if (!ossl_prov_is_running())
-        return NULL;
+    DIGEST_PROV_CHECK(provctx, aes);
```

Comment:
If you are going to use a macro please give it the right name for a cipher.

Reply:
yeah, I am not yet happy with this macro, but naming it right is kinda necessary.